test/db/cmd/cmd_i

```diff
@@ -4515,7 +4515,7 @@ RUN
 
 NAME=ik-relro
 FILE=bins/elf/analysis/x86-helloworld-gcc
-CMDS=ik info/elf.relro
+CMDS=ik elf.relro
```

Comment:
why this change?

Reply:
because before the `info` path was added by `rz_bin_object_new` in the following way:
```c
bin_obj->kv = plugin->get_sdb(); // pointer ownership is of the plugin
sdb_ns_set(bf->sdb, "info", bin_obj->kv);
bin_obj->kv = bf->sdb;
```
since that was one of the main causes of the un-freed pointers, then i decided to remove it since the usability is the same with the same errors.

Reply:
Ok. Not fully convincend though :D

But I don't care too much for SDB tbh and I hope it will go away one day.